### DISCUSSION TITLE
How to display modal over Ligthbox?

### DISCUSSION THREAD

**Discussion by sapkauskasm** [igordanchenko/yet-another-react-lightbox#42] [Q&A] [answered]
I want to delete an image from the light box, and before performing an action, I need to get confirmation modal.

The issue is that modal on top of lightbox is not interactive. 
Looks like backdrop or some overlay covers the screen.

Do you have any advice for workaround?
I can't figure out what the node portal binds to. 
My thinking:  If I could create Litghbox  portal on the same node as my modal <div id="portal-root">  maybe that would solve the issue?

 
<img width="1053" alt="Screenshot 2022-11-15 at 09 57 24" src="https://user-images.githubusercontent.com/20259412/201862003-57730ff0-c26b-454d-9ce3-7e8dc7a17e26.png">

**Comment by igordanchenko** [comment, marked as answer]
I'm afraid it's quite challenging to guess how exactly you are implementing a modal dialog and what particular issue you are facing by just looking at the above screenshot.

> I can't figure out what the node portal binds to.

The portal mounts as a last child of the `document.body`.

> Do you have any advice for workaround?

I can think of 2 ways to implement a confirmation dialog:

1) Use some modal dialog component (i.e., `react-modal`) and mount it at `z-index > 9999`

2) Manually render confirmation UI inside the lightbox (most likely via [render.controls](https://yet-another-react-lightbox.com/documentation#Render) + `position: absolute`)
